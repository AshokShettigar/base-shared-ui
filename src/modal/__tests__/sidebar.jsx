import { shallow } from 'enzyme';
import Sidebar from 'platform-common-ui/modal/sidebar';

describe('modal/sidebar', function() {
    it('should render the component', function() {
        const wrapper = shallow(<Sidebar>Sidebar content</Sidebar>);
        expect(wrapper).toMatchSnapshot();
    });
});
