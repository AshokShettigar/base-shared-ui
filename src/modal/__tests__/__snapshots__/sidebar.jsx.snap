// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal/sidebar should render the component 1`] = `
<div
  className="cn-modal-sidebar"
  data-component="Sidebar"
>
  Sidebar content
</div>
`;
